src/Page.js

```diff
@@ -590,31 +602,21 @@ Page.prototype.insertSiteNav = function (pageData) {
     throw new Error(`More than one <navigation> tag found in ${siteNavPath}`);
   } else if (siteNavDataSelector('navigation').length === 1) {
     const siteNavHtml = md.render(siteNavDataSelector('navigation').html().trim().replace(/\n\s*\n/g, '\n'));
-    const formattedSiteNav = formatSiteNav(siteNavHtml, this.src);
+    // Add Bootstrap padding class to rendered unordered list
+    const siteNavHtmlSelector = cheerio.load(siteNavHtml, { xmlMode: false });
+    siteNavHtmlSelector('ul').first().addClass('px-3');
+    const formattedSiteNav = formatSiteNav(siteNavHtmlSelector.html(), this.src);
     siteNavDataSelector('navigation').replaceWith(formattedSiteNav);
   }
   // Wrap sections
-  const wrappedSiteNav = `<div id="${SITE_NAV_ID}">\n${siteNavDataSelector.html()}\n</div>`;
-  const wrappedPageData = `<div id="${PAGE_CONTENT_ID}">\n${pageData}\n</div>`;
-
-  return `<div id="${FLEX_BODY_DIV_ID}">`
-    + `${wrappedSiteNav}`
-    + `${SITE_NAV_BUTTON_HTML}`
-    + `${wrappedPageData}`
-    + '</div>';
-};
-
-/**
- *  Inserts wrapper for page nav contents CSS manipulation
- */
-Page.prototype.insertPageNavWrapper = function (pageData) {
-  if (this.isPageNavigationSpecifierValid()) {
-    const wrappedPageData = `<div id="${PAGE_NAV_CONENT_WRAPPER_ID}">\n`
-                            + `${pageData}\n`
-                            + '</div>\n';
-    return wrappedPageData;
-  }
-  return pageData;
+  const wrappedSiteNav = `<nav id="${SITE_NAV_ID}" class="navbar navbar-light bg-transparent">\n`
+    + '<div class="sticky-top spacer-top viewport-height-90 scrollable slim-scroll">'
+    + `${siteNavDataSelector.html()}\n`
+    + '</div>\n'
+    + '</nav>\n';
+
+  return `${wrappedSiteNav}`
+    + `${pageData}`;
```

Comment:
Would ``return `${wrappedSiteNav}${pageData}`;`` work instead? 🙂

Reply:
Yes, it would work. However, within `wrappedSiteNav` there is a line break at the end of the string. So I wrote it in this manner to visually represent that wrappedSiteNav and `pageData` is separated by one line break.

Reply:
We can move the `\n` out of `wrappedSiteNav` to here, so that the visual representation is consistent.